website/static/js/pages/project-base-page.js

```diff
@@ -32,4 +32,41 @@ $(document).ready(function() {
     $.getJSON(node.urls.api, function(data) {    
         $('body').trigger('nodeLoad', data);
     });
+
+    var self = this;
+    var thresholdScrollPosition = 50;
```

Comment:
As per style guidelines, constants should be in all caps with underscores, e.g. `NAV_MAX_TOP_MARGIN`.